app/src/main/res/layout/spinner_layout.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <Spinner
+        android:id="@+id/spinnerAndroidVersions"
```

Comment:
Please consider formatting the code before it is sent for review. Use CTRL + ALT + L